### DISCUSSION TITLE
Inquiry Regarding Teacher Benefits on GitHub

### DISCUSSION THREAD

**Discussion by harisbinrehan-qbatch** [community/community#86047] [GitHub Education]
### Hi everyone,

I'm interested in accessing teacher benefits on GitHub with my current email. If I change my email in the future, how can I ensure a smooth transition while retaining these benefits?

**Comment by akashdip2001** [comment]
To retain teacher benefits on GitHub when you anticipate an email change, you can follow these guidelines:

1. **Apply with Current Email**: First, sign up for the GitHub Teacher Toolbox benefits using your current email address. You’ll need to provide proof of your educational status.

2. **Documentation**: Ensure all documentation linked to the benefits recognizes your name and not just the email. This will establish your identity, independent of the email address.

3. **Link Secondary Email**: If GitHub allows, add a secondary email to your account which will become your primary one later. This prepares your account for a smooth transition.

4. **Contact Support**: Before you change your email, reach out to GitHub support explaining your situation. Ask them for the best way to ensure a seamless transition of benefits to your new email.

5. **Update Email**: After consulting with support, update your email address in your GitHub account settings to your new email.

6. **Verify Benefits Transfer**: Check to ensure that your benefits are still active after the email change. Contact support if there are any issues.

7. **Update Academic Records**: If you are switching schools or jobs, update your academic records on GitHub to reflect your new affiliation.

8. **Regular Check-In**: Keep an eye on your benefits expiration and any emails from GitHub regarding the status of your benefits.

By taking these steps, you should be able to change your email without losing access to your teacher benefits on GitHub. As policies and procedures may vary, always check the latest guidance from GitHub’s support resources.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by harisbinrehan** [reply]
Do teacher benifits include GitHub Copilot?
